src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_help.py

```diff
@@ -419,12 +419,32 @@
 
 helps['functionapp devops-build'] = """
 type: group
-short-summary: Azure Function specific integration with Azure DevOps.
+short-summary: (DEPRECATED) Azure Function specific integration with Azure DevOps.
 """
 
 helps['functionapp devops-build create'] = """
 type: command
+short-summary: (DEPRECATED) Create an Azure DevOps pipeline for a function app.
```

Comment:
add a pointer to the new command, something like
short-summary: (DEPRECATED) Create an Azure DevOps pipeline for a function app. Please use the devops-pipeline create command.